src/lib/apiTypesV2/Entity.cpp

```diff
@@ -81,27 +81,39 @@ std::string Entity::render(ConnectionInfo* ciP, RequestType requestType, bool co
 
   if ((errorCode.description == "") && ((errorCode.error == "OK") || (errorCode.error == "")))
   {
-    std::string out = "{";
-
-    out += JSON_VALUE("id", id);
-    out += ",";
-    out += JSON_STR("type") + ":" + ((type != "")? JSON_STR(type) : "null");
-
-    if (attributeVector.size() != 0)
+    if (renderMode == "values")
     {
+      std::string out = "[";
+      if (attributeVector.size() != 0)
+      {
+        out += attributeVector.toJson(true, false, renderMode, ciP->uriParam["attrs"]);
+      }
+      out += "]";
+      return out;
+    }
+    else
```

Comment:
Fixed in 1a99b1b7525c63f4b01aba431a07e16a60703fa1 (actually the else is needed, as there is common code).